proto/aquila/event.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+option ruby_package = "Tucana::Aquila";
+
+package aquila;
+
+// Event that gets admitted by an action
+message Event {
+  // Id of Event type
+  int32 event_type = 1;
```

Comment:
I’m not so sure if you have the database id in Aquila. But it definitely makes more sense. Maybe its necessary to implement the return of the ids in Sagittarius.